public/styles.css

```diff
@@ -206,6 +206,12 @@ table > tbody > tr > td {
     border-radius: 8px;			
     pointer-events: none;			
 }
+tr:hover td { 
+    cursor: pointer;
+}
+th:hover {
+    cursor: grab;
```

Comment:
I think this should be pointer not grab, because it's something that's clickable not something that's draggable.

Reply:
(But great that the cursor now changes!)